doc/man/man8/dmg.8

```diff
@@ -0,0 +1,354 @@
+.TH dmg.test 1 "25 March 2020"
+.SH NAME
+dmg.test \- Administrative tool for managing DAOS clusters
```

Comment:
Is `dmg.test` correct?

Reply:
Hmm that's really odd. I don't know why its adding .test there. Ill have to look through the flags library to see where exactly its pulling this name from and see if we can override it.